site/content/blog/eth2-dev-chain.md

```diff
@@ -0,0 +1,201 @@
+---
+title: Setting up private Ethereum 2.0 development chain
+date: 2020-03-26T09:00:00-00:00
+image: /images/blog/ethereum20-validator.png
+draft: false
+keywords:
+- ethereum 2.0
+- proof of staking
+- dev
+- lighthouse
+- chainguardian
+description: Learn how to setup eth2 development chain using lighthouse client. 
+---
+
+> Disclaimer: This article is highly technical intended for developers and eth2 enthusiasts.
+
+### Why would I setup private eth2 chain?
+
+If you had any experience with development on eth1, you probably already know answer to this question so 
+feel free to skip ahead.
+
+At the moment, eth2 has a bunch short lived single client testnets and [one long lived](https://prylabs.net/) with
```

Comment:
You are linking Prysmatic 2x times and this is kinda hidden linking :D
Rephrase so you only link once the name i.e. `At the moment, eth2 has a bunch of short lived single client testnets and one long lived which is Prysmatic's Sapphire Testnet](https://prylabs.net/). However, real stable client testnet is yet to come.`